CMakeLists.txt

```diff
@@ -4,6 +4,10 @@ project(mppic)
 add_definitions(-DXTENSOR_ENABLE_XSIMD)
 add_definitions(-DXTENSOR_USE_XSIMD)
 
+set(XTENSOR_USE_TBB 0)
```

Comment:
Wouldn't 1 be enabling? Does this change anything?

Reply:
I'm not sure, but i would prefer to keep things more one threaded till we reduce jitter problem. One thread - more predictable behavior